src/components/vue-form-builder.vue

```diff
@@ -149,68 +149,104 @@
     <!-- Inspector -->
     <b-col class="overflow-hidden h-100 p-0 inspector-column">
       <b-card no-body class="p-0 h-100">
-        <b-card-header>{{ $t('Inspector') }}</b-card-header>
-
+        <b-card-header>
+          {{ $t('Inspector') }}
+        </b-card-header>
         <b-card-body class="p-0 h-100 overflow-auto">
+
           <b-button
-            v-b-toggle.configuration
```

Comment:
There is already a PR for this change that just needs the copy updated: https://github.com/ProcessMaker/screen-builder/pull/263. And the implementation here doesn't seem to work as expected (the variable accordion is actually in the "open" state but empty):

<img width="882" alt="Screen Shot 2019-07-02 at 1 51 04 PM" src="https://user-images.githubusercontent.com/7561061/60534753-8b8e4c00-9cd0-11e9-8a95-73a2233d9e6c.png">

Reply:
This requirement comes from this ticket:
https://github.com/ProcessMaker/processmaker/issues/2001

In which requires the read only property must be inside the Variable panel:
![image](https://user-images.githubusercontent.com/8028650/60603469-d1f0b300-9d83-11e9-8987-60c5907eac3f.png)

Reply:
I assume there was a mixup and @marcoAntonioNina grabbed https://github.com/ProcessMaker/processmaker/issues/2001 and started doing work in this repo, even though there have already been tickets and PRs created for that issue in this repo.